#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_add_rsync.dpatch by  <mika@grml.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add support for rsync

@DPATCH@

--- a/hooks/live	Thu Jul 17 14:16:44 2008 +0200
+++ b/hooks/live	Thu Jul 17 14:17:05 2008 +0200
@@ -86,6 +86,8 @@ manual_add_modules ext3
 # Filesystem: jffs2
 manual_add_modules jffs2
 
+copy_exec /usr/bin/rsync /bin
+
 # Filesystem: squashfs
 copy_exec /sbin/losetup /sbin
 manual_add_modules loop
